LullabyRecipe/Views/Kitchen/StudioView.swift

```diff
@@ -0,0 +1,123 @@
+    @State private var showingAlert = false
+    @State private var selectedBaseSound: Sound = Sound(id: 0,
+                                                        name: "",
+                                                        soundType: .base,
+                                                        audioVolume: 0.8,
+                                                        imageName: "")
+    @State private var selectedMelodySound: Sound = Sound(id: 0,
+                                                          name: "",
+                                                          soundType: .melody,
+                                                          audioVolume: 1.0,
+                                                          imageName: "")
+    @State private var selectedNaturalSound: Sound = Sound(id: 0,
+                                                           name: "",
+                                                           soundType: .natural,
+                                                           audioVolume: 0.4,
+                                                           imageName: "")
+    @State var userName: String = ""
+    @State private var textEntered = ""
+
+    let baseAudioManager = AudioManager()
+    let melodyAudioManager = AudioManager()
+    let naturalAudioManager = AudioManager()
+
+    private var items = ["BASE", "MELODY", "NATURAL"]
+    var body: some View {
+        VStack {
+            SelectImage()
+            CustomSegmentControlView(items: items, selection: $select)
+            switch select {
+            case 1:
+                SoundSelectView(sectionTitle: "Melody",
+                                soundType: .melody)
+            case 2:
+                SoundSelectView(sectionTitle: "Natural Sound",
+                                soundType: .natural)
+            default:
+                SoundSelectView(sectionTitle: "Base Sound",
+                                soundType: .base)
+
+            }
+        }
+    }
+
+    @ViewBuilder
+    func SoundSelectView(sectionTitle: String,
+                         soundType: SoundType) -> some View {
+        VStack(spacing: 15) {
+
+            HStack {
+                Text("볼륨조절 컴포넌트")
+            }
+
+            ScrollView(.vertical,
+                       showsIndicators: false) {
+                HStack(spacing: 30) {
+
+                    switch soundType {
+                    case .base:
+                        RadioButtonGroup(selectedId: soundType.rawValue,
+                                         items: baseSounds) { baseSelected in
+                            selectedBaseSound = baseSelected
+                            // play music
+
+                            if selectedBaseSound.name == "Empty" {
+                                baseAudioManager.stop()
+                            } else {
+                                baseAudioManager.startPlayer(track: baseSelected.name)
+                            }
```

Comment:
selectedBaseSound에 baseSelected를 할당하였으므로 if else 구문에서 사용되는 프로퍼티를 selectedBaseSound로 통일하는 것은 어떨까요? 
```suggestion
                            selectedBaseSound = baseSelected
                            // play music

                            if selectedBaseSound.name == "Empty" {
                                baseAudioManager.stop()
                            } else {
                                baseAudioManager.startPlayer(track: seletedBaseSound.name)
                            }
```

Reply:
Good!